packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -8,325 +8,14 @@
 /* eslint-env node */
 
 // For more information on these settings, see https://webpack.js.org/configuration
-import fs from 'fs'
-import path, {resolve} from 'path'
-
-import webpack from 'webpack'
-import WebpackNotifierPlugin from 'webpack-notifier'
-import CopyPlugin from 'copy-webpack-plugin'
-import {BundleAnalyzerPlugin} from 'webpack-bundle-analyzer'
-import LoadablePlugin from '@loadable/webpack-plugin'
-import SpeedMeasurePlugin from 'speed-measure-webpack-plugin'
-import {createModuleReplacementPlugin, PwaKitConfigPlugin} from './plugins'
+const fs = require('fs')
+const path = require('path')
 
 const projectDir = process.cwd()
-const sdkDir = path.resolve(path.join(__dirname, '..', '..', '..'))
-
-const pkg = require(resolve(projectDir, 'package.json'))
-const buildDir = resolve(projectDir, 'build')
-
-const production = 'production'
-const development = 'development'
-const analyzeBundle = process.env.MOBIFY_ANALYZE === 'true'
-const mode = process.env.NODE_ENV === production ? production : development
-const DEBUG = mode !== production && process.env.DEBUG === 'true'
-const CI = process.env.CI
-
-if ([production, development].indexOf(mode) < 0) {
-    throw new Error(`Invalid mode "${mode}"`)
-}
-
-const entryPointExists = (segments) => {
-    for (let ext of ['.js', '.jsx', '.ts', '.tsx']) {
-        const p = path.resolve(projectDir, ...segments) + ext
-        if (fs.existsSync(p)) {
-            return true
-        }
-    }
-    return false
-}
+const projectWebpackPath = path.resolve(projectDir, 'webpack.config.js')
```

Comment:
@vcua-mobify Currently our pwa template does _not_ have this webpack config file. But let's reconsider that, because the template does have other config files, even when some are only one-liner. I think to be consistent (and also easier to discover by the end users), we need to create this webpack.config.js in the pwa too.

The pwa template currently has these other configs, which has a pattern of importing from `pwa-kit-build`:
- [babel.config.js](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/6be1fc698ce9a9df58277cd58e45f66accd4b098/packages/pwa/babel.config.js#L7)
- [jest.config.js](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/6be1fc698ce9a9df58277cd58e45f66accd4b098/packages/pwa/jest.config.js#L1-L8)